src/main/java/org/sciborgs1155/robot/Constants.java

```diff
@@ -17,4 +18,9 @@
 public class Constants {
   public static final Measure<Time> PERIOD = Seconds.of(0.02); // roborio tickrate (s)
   public static final double DEADBAND = 0.1;
+  public static final double MAX_RATE =
+      DriveConstants.MAX_ACCEL.baseUnitMagnitude()
+          / DriveConstants.MAX_ANGULAR_SPEED.baseUnitMagnitude();
+  public static final double SLOW_SPEED = 0.33;
+  public static final double FULL_SPEED = 1.0;
```

Comment:
a bit of a misleading name (slow_speed too) since it's a scalar, not the actual speed